ansible_collections/arista/avd/roles/eos_config_deploy_cvp/tasks/main.yml

```diff
@@ -5,9 +5,9 @@
   inventory_to_container:
     inventory: '{{ inventory_file }}'
     container_root: '{{ container_root }}'
-    configlet_dir: '{{playbook_dir}}/intended/configs'
+    configlet_dir: '{{inventory_dir}}/intended/configs'
     configlet_prefix: '{{ configlets_prefix }}'
-    destination: '{{inventory_dir}}/intended/structured_configs/cvp/{{inventory_hostname}}.yml'
+    destination: '{{inventory_dir}}/intended/structured_configs/cvp/{{inventory_hostname}}_2.yml'
```

Comment:
from @titom73 "because we do a register, this file is not required. But it is usefull for debug.
Previous version overwritte it with next task. So I use _2 to keep a trace
I can rename it to _configlets"